internals/models.py

```diff
@@ -970,6 +970,8 @@ def __notify_feature_subscribers_of_changes(self, is_update):
       new_val = getattr(self, prop_name, None)
       old_val = getattr(self, '_old_' + prop_name, None)
       if new_val != old_val:
+        if new_val == '' and old_val is None:
+          continue
         new_val = convert_enum_int_to_string(prop_name, new_val)
         old_val = convert_enum_int_to_string(prop_name, old_val)
```

Comment:
@jrobbins,as suggested by you, I have modified the function  `__notify_feature_subscribers_of_changes()`. I tested it also. It seems to fix the problem of None vs Empty String in diffs.

Reply:
However, there are some fields in the diff with the following old and new values:
`old_val` : None
`new_val`: False

Are such fields supposed to be NOT present in diffs ?